### ISSUE TITLE
Nitwits With 0 Intelligence Not Accepting Professions

### ISSUE THREAD

**Issue by PfluorescentZebra** [TangoTek/TekTopia-Community#176]
**Describe the bug**
Nitwits are not longer being converted into hardworking, useful Tektopians after 1.0.18 update.

**To Reproduce**
1. Click on nitwit with profession token
2. No change happens

**Expected behavior**
Nitwit should be converted to the profession and the token should disappear from player inventory.

** Server/Location **
What server were you playing on? All of them, Reports from 1 & 2
What are the world coordinates of your town hall marker? 2680 116 -8673

**Additional context**
Paspinall, Steve_Jobz, Pazza404, and myself have experienced this but were successful prior to the update.

**Comment by Shamshalabim** [comment]
I've experienced the opposite... No issues in this area, and several villagers converted on Main Server 1

**Comment by Hedkracka** [comment]
I just changed one to a LJ on first click

**Comment by sclough3014** [comment]
I've seen this too.  If I right-click on my nitwit with anything other than the token it opens his inventory, if I try to use the token on him, nothing happens, the UI doesn't open, and the token doesn't get applied

**Comment by JohnyDL** [comment]
Might be the token broke after the 1.0.18 update?

**Comment by LinaBoeckwurm** [comment]
Same over here. It doesn't open the inventory, nor does it convert.

**Comment by sclough3014** [comment]
I don't think it's the token that is broken, since I can still convert nomads.  I think the nitwit is what's broken

**Comment by LinaBoeckwurm** [comment]
I had the thought that the fact that Tango recently changed something about the fact that nitwits now gain levels in the job that they get converted to according to how well studied they are. 
I didn't have a library or a school in my villager, so the kid grew up without any knowledge.
I've since build a library in my village, and my nitwit has been studying in it it. 
After it got a bit of knowledge, I could convert it to a druid. 
Granted, this after restarting my single player world several times, so there could also be other reasons that fixed it, but does this align with other people's experiences as well?
Were @PfluorescentZebra and @sclough3014's nitwits unschooled?

**Comment by ekkelos** [comment]
i'm having the same problem i tried diverent profecions on my nitwit: farmer, miner and guard and non of them are accepted by the nitwit

**Comment by sclough3014** [comment]
My nitwit was unschooled, but it's tough on early level villages to require education for the nitwits, schools and libraries are expensive

**Comment by Geex204** [comment]
I have this as well but haven't had any nitwits before the update so no idea if that changed then.

**Comment by LinaBoeckwurm** [comment]
I agree @sclough3014 that what should actually happen is that the nitwit just gains level 1 in the chosen profession, just like normally would happen when you convert a nomad into a profession that they hadn't acquired yet. But I was trying to pinpoint the cause of the issue so it was easier to fix ^.^

**Comment by JohnyDL** [comment]
@LinaBoeckwurm I'm inclined to agree with you it might be unschooled (0 knowledge) Nitwits get 0 levels in the job and so don't covert or use up the token and this needs fixing.

If you're experiencing this bug can you screenshot your broken nitwit stats when you're trying to convert them?

**Comment by ThomasBerends** [comment]
Confirmed this. It's nitwits with zero intelligence. Here's 3 of them, all of them don't accept my profession tokens.

![2019-06-02_20 43 07](https://user-images.githubusercontent.com/1807910/58765804-1f0c1a00-8577-11e9-9e1f-04cec4f876d7.png)
![2019-06-02_20 42 55](https://user-images.githubusercontent.com/1807910/58765805-1fa4b080-8577-11e9-978e-3f95145e6bd8.png)
![2019-06-02_20 42 59](https://user-images.githubusercontent.com/1807910/58765806-1fa4b080-8577-11e9-9ef1-d62c63496e08.png)

**Comment by ThomasBerends** [comment]
@PfluorescentZebra Can you update the title for this to 'Nitwits with zero intelligence don't accept profession tokens' ?
